src/components/summary/_summary.scss

```diff
@@ -208,4 +208,36 @@ $hub-row-spacing: 1.3rem;
       }
     }
   }
+
+  &__group {
+    &--card {
+      border: 1px solid var(--ons-color-borders-light);
+      .ons-summary__items {
+        padding: 0 1.25rem;
+        .ons-summary__item:not(:last-child) {
+          border-bottom: 1px solid var(--ons-color-borders-light);
```

Comment:
Just wondering what IxD thinks of using light grey here to separate sections. The rest of the summary component uses the dark grey for borders